backend/src/main/java/com/crm/dto/request/CarRequest.java

```diff
@@ -0,0 +1,53 @@
+package com.crm.dto.request;
+
+import com.crm.exception.ErrorDict;
+import com.crm.model.db.CarTypeEntity;
+import lombok.*;
+
+import javax.validation.constraints.*;
+
+@Getter
+@Setter
+@Builder
+@EqualsAndHashCode
+@NoArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
```

Comment:
Łatwiej będzie użyć @Data
"Equivalent to {@ code @ Getter @ Setter @ RequiredArgsConstructor @ ToString @ EqualsAndHashCode}."

Reply:
To nie zadziała z Builderem, wywala błędy. Builder wymaga AllArgsConstructor, który tworzy konstruktor dla wszystkich pól, zaś RequiredArgsCostructor tworzy bodajże tylko dla pól typu final. Zostawiłem jak było, gdyż w innych klasach powtarza się ten motyw.